### ISSUE TITLE
Accessibility issue: Elements must meet minimum color contrast ratio thresholds on /notebook-ci-testing/notebooks/**/*.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#111]
## What
An accessibility scan flagged 8 elements on https://spacetelescope.github.io/notebook-ci-testing/notebooks/**/*.html because elements must meet minimum color contrast ratio thresholds. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/color-contrast?application=playwright.

The following elements need attention:

- `<strong>File not found</strong>` (selector: `p:nth-child(2) > strong`)
- `<p>
        The site configured at this address does not
        contain the requested file.
      </p>` (selector: `p:nth-child(3)`)
- `<p>
        If this is your site, make sure that the filename case matches the URL
        as well as any file permissions.<br>
        For root URLs (like <code>http://example.com/</code>) you must provide an
        <code>index.html</code> file.
      </p>` (selector: `p:nth-child(4)`)
- `<code>http://example.com/</code>` (selector: `code:nth-child(2)`)
- `<code>index.html</code>` (selector: `code:nth-child(3)`)
- `<p>
        <a href="https://help.github.com/pages/">Read the full documentation</a>
        for more information about using <strong>GitHub Pages</strong>.
      </p>` (selector: `p:nth-child(5)`)
- `<a href="https://help.github.com/pages/">Read the full documentation</a>` (selector: `a[href$="pages/"]`)
- `<strong>GitHub Pages</strong>` (selector: `p:nth-child(5) > strong`)

To fix this, ensure the contrast between foreground and background colors meets wcag 2 aa minimum contrast ratio thresholds.

Specifically:

Fix any of the following:
-   Element has insufficient color contrast of 3.85 (foreground color: #797979, background color: #f1f1f1, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
